<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="input_ip">
           <property name="maxLength">
            <number>32769</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="botao_connect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="botao_disconect">
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="horizontalSlider_min">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_min">
             <property name="text">
              <string>Min</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_min"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSlider" name="horizontalSlider_max">
           <property name="maximum">
            <number>99</number>
           </property>
           <property name="value">
            <number>35</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="max">
             <property name="text">
              <string>Max</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_max">
             <property name="value" stdset="0">
              <double>35.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Timing (ms) </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_timer">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="sliderPosition">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="value" stdset="0">
            <double>1000.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>disconnected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_min</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_max</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_timer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
